skrifa/src/outline/autohint/outline.rs

```diff
@@ -197,6 +202,35 @@ impl Outline {
         self.points.clear();
         self.contours.clear();
     }
+
+    pub fn to_path(
+        &self,
+        style: PathStyle,
+        pen: &mut impl OutlinePen,
+    ) -> Result<(), path::ToPathError> {
+        for contour in &self.contours {
+            let Some(points) = self.points.get(contour.range()) else {
+                continue;
+            };
+            #[inline(always)]
+            fn to_contour_point(point: &Point) -> path::ContourPoint<F26Dot6> {
+                const FLAGS_MAP: [PointFlags; 3] = [
+                    PointFlags::on_curve(),
+                    PointFlags::off_curve_quad(),
+                    PointFlags::off_curve_cubic(),
+                ];
+                path::ContourPoint {
+                    x: F26Dot6::from_bits(point.x),
+                    y: F26Dot6::from_bits(point.y),
+                    flags: FLAGS_MAP[(point.flags & Point::CONTROL) as usize],
```

Comment:
I initially expected `flags: PointFlags::from_bits(point.flags)` and was confused. autohint::outline::Point has `flags: u8` which looks an awful lot like it should fit there. 

At the absolute minimum I suggest comments here and in the autohint Point flags field to clarify they arne't the same. 

Unless there is a good reason not to I suggest we make autohint Point flags a newtype like glyf::PointFlags. Presumably it could then have a conversion method to produce glyf PointFlags and then this could would flow very idiomatically.

Reply:
Changed to just use the existing `glyf::PointFlags` entirely. Added a couple new "markers" and some new methods to support this.